.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
 # Install SDK license so Android Gradle plugin can install deps.
 - mkdir "$ANDROID_HOME/licenses" || true
 - echo "d56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
+- echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" >> "$ANDROID_HOME/licenses/android-sdk-license"
```

Comment:
Shouldn't this replace the line above?

Reply:
No that didn't work. The first one is still required for the emulator I believe.